### ISSUE TITLE
Simpler path bar

### ISSUE THREAD

**Issue by tintou** [elementary/files#2033]
### Problem

The current code for the path bar is really non-obvious and will be difficult to move to GTK4

### Proposal

Have a less complicated path bar.
In macOS the path bar is even hidden by default, while I don't think that it is what we want here, having it styled with simple arrows would make it simpler and easier to maintain.

### Prior Art (Optional)

Finder: ![image](https://user-images.githubusercontent.com/745535/164322333-55091f13-7f81-411e-a8ab-ce4e10ee98f6.png)

**Comment by danrabbit** [comment]
Yeah I think the slash style like in GNOME Files Gtk4 port looks nice too

**Comment by jeremypw** [comment]
Also need to decide how to incorporate the Searchbar.

**Comment by hanaral** [comment]
@jeremypw Would it be _too_ wrong to have a search bar like appcenter or music? If there isn't going to be a text based pathbar then the next most obvious thing is a permanent one.

**Comment by jeremypw** [comment]
It would certainly improve discoverability to have it permanently visible although it may increase the minimum width. I guess it need not be too large as search terms are generally short. Other possibilities would be a Code-like search bar in a revealer, or having it in a Stack with the path bar.  Anyway it would be a separate widget.

**Comment by PlatonB** [comment]
> Yeah I think the slash style like in GNOME Files Gtk4 port looks nice too

Pathbar in Gnome seems bad to me because there is no way to copy the current directory path.
